node/resolvers/catalog/index.ts

```diff
@@ -196,24 +196,33 @@ export const queries = {
     return map(resolveCategoryFields, categories)
   },
 
-  search: async (_, data, { vtex: ioContext }: ColossusContext, info) => {
-    const { map, query, rest } = data
-    const facetsMap = map.split(',').slice(0, query.split('/').length).join(',')
+  search: async (_, data, colossusContext: ColossusContext, info) => {
```

Comment:
Why you need this? ioContext isn't enough?

Reply:
The tslint was complaining because of the argument type of the other queries, but I can change back to how it was.

Reply:
I prefer. tslint know nothing. :trollface: